keyboards/dweebsmini/keymaps/default/keymap.c

```diff
@@ -0,0 +1,86 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  COLEMAK,
+  DVORAK
+};
+
+#define LOWER  MO(_LOWER)
+#define RAISE  MO(_RAISE)
+#define ADJUST MO(_ADJUST)
+#define CTLESC CTL_T(KC_ESC)
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_QWERTY] = LAYOUT(
+   KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,  KC_LPRN,           KC_RPRN,  KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC, \
+   CTLESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,  KC_PGUP,           KC_PGDN,  KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, \
+   KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                               KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, \
+                              ADJUST,  KC_LGUI, RAISE, KC_SPACE,          KC_ENTER, LOWER,   KC_RALT,  KC_RCTL  \
+),
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
I am leaving it the way it is